app/src/main/java/com/animal_villa/dto/Player.kt

```diff
@@ -1,4 +1,11 @@
 package com.animal_villa.dto
 
+import androidx.room.Entity
+
+@Entity
 class Player {
+    var Status:Int=100;
+    var Money=0;
+    var Energy=30;
+    var CurrentLocation=0;
```

Comment:
variable names should be lowerCamelCase in Kotlin.

This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.